deployment/src/main/java/io/quarkiverse/quinoa/deployment/packagemanager/PackageManagerInstallConfig.java

```diff
@@ -69,4 +96,4 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(enabled, nodeVersion, nodeDownloadRoot);
```

Comment:
You need to update the equals and hashcode

Reply:
Updated